cypress/support/TestCases/VerifyLoginWithMissingCredentials.js

```diff
@@ -0,0 +1,14 @@
+import {LoginPage} from "../PagesObjects/LoginPage"
+export default function verifyLoginWithMissisingCredentials() {
+  const login_page  = new LoginPage()
+  
+  login_page.clickLoginButton()
```

Comment:
you dont need to write a function for every specific action better to write a function for a flow that will repeat itself across different test cases

Reply:
Resolved